### DISCUSSION TITLE
Qulacs

### DISCUSSION THREAD

**Discussion by camillerievan** [zapatacomputing/orqviz#60] [General]
Does this package work with Qulacs?

**Comment by mstechly** [comment]
It works with any arbitrary cost function you use, so if you define a cost function which uses qulacs, it works with qulacs :)
In this notebook we have an example of how it can work with Qulacs, more specifically with an adapter between Orquestra stack and Qulacs: https://github.com/zapatacomputing/orqviz/blob/60c415670a2af5cd933062334924fcec51fc87ad/docs/examples/orqviz_tutorial_orquestra.ipynb#L198
